common/flatpak-run-wayland.c

```diff
@@ -217,5 +284,27 @@ flatpak_run_add_wayland_args (FlatpakBwrap *bwrap,
                               NULL);
       flatpak_bwrap_add_runtime_dir_member (bwrap, wayland_display);
     }
+
+  /* If inherit-wayland-socket is not set, unset WAYLAND_SOCKET unconditionally
+   * without checking the validity of the value of WAYLAND_SOCKET. */
+  if (!inherit_wayland_socket)
+    flatpak_bwrap_unset_env (bwrap, "WAYLAND_SOCKET");
+
+  fd = get_wayland_socket_fd ();
+  if (fd >= 0)
+    {
+      if (inherit_wayland_socket)
+        flatpak_bwrap_add_fd (bwrap, fd);
+      else
```

Comment:
```suggestion
      if (inherit_wayland_socket)
        {
          flatpak_bwrap_add_fd (bwrap, fd);
        }
      else
```
Use braces for the if branch when the else branch uses braces.